packages/grafana-data/src/types/config.ts

```diff
@@ -110,7 +110,6 @@ export interface CurrentUserDTO {
   email: string;
   name: string;
   theme: string; // dark | light | system
-  orgCount: number;
```

Comment:
These are public interfaces used by plugins. Are we sure we want to remove this property? It will be a breaking change for any plugin that uses this. (If there are any plugins out there that do use it.)